### PR TITLE
Put the entire git message

### PR DESCRIPTION
**Pull request by sileht**
When only one commit is part of the pull request. This change will send
the entire commit message instead of duplicating the title.

### CODE REVIEW

File: git_pull_request/__init__.py [hunk 1]
```diff
@@ -221,15 +221,19 @@ def git_pull_request(target_remote=None, target_branch=None,
...
-                       output=True))
+        if not message:
+            message = _run_shell_command(
+                ["git", "log", "--format=%s",
+                 target_remote + "/" + target_branch + ".." + branch],
+                output=True)
 
         title = title or get_title_from_git_log(message, branch)
 
+        if len(message.split('\n')):
```
Comment (jd) [inline git_pull_request/__init__.py]: I don't get it. If there's multiple lines in the message there's multiple commit… so why doing this?

also it'd be clearer to add it in the `if not message` after L228.

Or I just miss coffee?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 31fd2d365e613ed8ad2513b83130807faaa81667

**Comment by sileht** [comment]
I have rewrote it in an more understandable way.

**Comment by jd** [comment]
> I have rewrote it in an more understandable way.

Haha thank you for telling me I'm stupid so gently :-D

**Comment by jd** [comment]
You forgot to update the test though :)

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 10a5d80059fb0412b8c74f49c4beef658d795aa1

**Comment by jd** [comment]
Test failed, might be a different git version if it works locally? just thinking

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 25b275f4c019277288af9dd8ad605939aa695ad8

**Comment by jd** [comment]
nop. 💃

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 3208f34a8f143d8afa50da3275e9e6cfce37fcf6

**Pull request merged**